src/Algebra/Construct/NaturalChoice/MinOp.agda

```diff
@@ -39,13 +39,13 @@ open import Relation.Binary.Reasoning.Preorder preorder
 
 x⊓y≤x : ∀ x y → x ⊓ y ≤ x
 x⊓y≤x x y with total x y
-... | inj₁ x≤y = ≤-respˡ-≈ (Eq.sym (x≤y⇒x⊓y≈x x≤y)) refl
+... | inj₁ x≤y = reflexive (x≤y⇒x⊓y≈x x≤y)
 ... | inj₂ y≤x = ≤-respˡ-≈ (Eq.sym (x≥y⇒x⊓y≈y y≤x)) y≤x
```

Comment:
I think I agree with your choice (as with L47-L48 below), but am also conscious that a certain *kind* of 'symmetry' has been lost between the two branches here. But on balance, happy with the 'improvement'. YMMV!